website/docs/quickstart/security.md

```diff
@@ -0,0 +1,510 @@
+---
+title: Secure Your Fluss Cluster
+sidebar_position: 1
+---
+
+<!--
+ Copyright (c) 2025 Alibaba Group Holding Ltd.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+#  Secure Your Fluss Cluster in Minutes
+This guide demonstrates how to secure your Fluss cluster using two practical examples:
+1. Securing a Fluss Cluster within a Department with Different Roles
+2. Enabling Multi-Tenant Isolation in a Fluss Cluster
+
+These scenarios will help you understand how to configure authentication and authorization, manage access control, and implement data isolation in real-world use cases.
+
+## Example 1: Secure Your Fluss Cluster Within a Department Using Role-Based Access Control
```

Comment:
Actually, this is not RBAC, because you just create 3 different users, not roles. 

Change the title to `Example 1: Secure Fluss with Different Roles`